pandokia/top_level.html

```diff
@@ -63,6 +63,8 @@
 <br/>
 
 <a href="CGINAME?query=expected">Expected tests summary</a>
+<p>
+<a href="https://glitch.etc.stsci.edu/jwst/test/reports">JWST ETC test reports (includes log files)</a>
```

Comment:
Test report means something specific in Pandokia; if we could call this "JWST ETC test meta-reports" it would make me happier.

This is a non-blocking comment.

Reply:
Can Do!!